### PR TITLE
CANVAS-74 - Changed hrefs to iframes.

### PR DESCRIPTION
**Pull request by kilometerskang**

### CODE REVIEW

File: src/client/components/App/index.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Iframe {
+    width: 800px;
```
Comment (rlorenzo) [inline src/client/components/App/index.css:2]: @kilometerskang Having fixed px ratios is not great for mobile viewing. Can you try to make the iframe responsive? 

See https://www.w3docs.com/snippets/css/how-to-create-a-responsive-iframe-with-css.html

File: src/client/components/App/index.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Iframe {
+    position: absolute;
+    width: 99%;
+    height: 100%;
```
Comment (kilometerskang) [inline src/client/components/App/index.css:4]: I made the width 99% because with 100%, the iframe had a tiny bit of scroll so it felt choppy.

**Comment by rlorenzo** [comment]
@kilometerskang I am comparing the functionality to what we have currently and we need to make a few changes:

* Add "Library resources feedback" to both "Research guide" and "Course reserves"
* Make "Request library reserves open into a new window. When you make the window open into a new window by _blank, remember to add rel="noopener roreferrer". See https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

**Comment by rlorenzo** [comment]
@kilometerskang I checked out the latest branch and I still don't see  "Library resources feedback" in the Research guide tab. Instead I see "Request library reserve". "Request library reserve" should only appear in the "Course reserve" tab.

Then I logged in as a student and I see "Request library reserve", which should only be displayed to Instructors or Admins. Also, as a student, if there are no "Course reserves" that tab shouldn't even show up for me.

**Comment by kilometerskang** [comment]
> @kilometerskang I checked out the latest branch and I still don't see "Library resources feedback" in the Research guide tab. Instead I see "Request library reserve". "Request library reserve" should only appear in the "Course reserve" tab.
> 
> Then I logged in as a student and I see "Request library reserve", which should only be displayed to Instructors or Admins. Also, as a student, if there are no "Course reserves" that tab shouldn't even show up for me.

**Comment by kilometerskang** [comment]
> @kilometerskang I checked out the latest branch and I still don't see "Library resources feedback" in the Research guide tab. Instead I see "Request library reserve". "Request library reserve" should only appear in the "Course reserve" tab.
> 
> Then I logged in as a student and I see "Request library reserve", which should only be displayed to Instructors or Admins. Also, as a student, if there are no "Course reserves" that tab shouldn't even show up for me.

How should the "Library resources feedback" page show up? Should it still be an iframe, even though "Research Guide" will already have a different iframe for the Spring Share tool too?

**Comment by kilometerskang** [comment]
> > @kilometerskang I checked out the latest branch and I still don't see "Library resources feedback" in the Research guide tab. Instead I see "Request library reserve". "Request library reserve" should only appear in the "Course reserve" tab.
> > Then I logged in as a student and I see "Request library reserve", which should only be displayed to Instructors or Admins. Also, as a student, if there are no "Course reserves" that tab shouldn't even show up for me.
> 
> How should the "Library resources feedback" page show up? Should it still be an iframe, even though "Research Guide" will already have a different iframe for the Spring Share tool too?

or maybe I can have the feedback link under "Research Guide" link to the iframe under "Course Reserves"

**Comment by rlorenzo** [comment]
@kilometerskang Yeah, that would be confusing to overwrite the iframe of the research guide.. How would you get back to the Research guide? What if we just make both buttons open into a new window?

**Pull request merged**